models/rose/README.rst

```diff
@@ -0,0 +1,29 @@
+###########
+ROSE README
+###########
```

Comment:
Support for this model was frozen in Lanai ... it is not Manhattan-compliant.

Reply:
Tomoko moved on to TIE-GCM, maybe we want to steer people that way ...

Reply:
We discussed in standup that we should be agnostic about which model(s) users should attempt to use, so we decided against instructing people to switch from ROSE to TIE-GCM.